website/static/worker.js

```diff
@@ -22,9 +22,8 @@ var prettier = index; // eslint-disable-line
 var parsersLoaded = {};
 
 self.onmessage = function(message) {
-  const options = message.data.options || {
-    parser: "babylon"
-  };
+  var options = message.data.options || {};
```

Comment:
This change is for backwards compatibility with _really_ old prettier links that didn't have `parser` set in options.